prow/github/client.go

```diff
@@ -454,7 +456,7 @@ func (c *Client) RemoveLabel(org, repo string, number int, label string) error {
 type MissingUsers []string
 
 func (m MissingUsers) Error() string {
-	return fmt.Sprintf("cannot assign %s", strings.Join(m, ", "))
+	return fmt.Sprintf("cannot assign or request review from %s", strings.Join(m, ", "))
```

Comment:
Are we attempting to assign or are we attempting to request a review?

This change makes the error more confusing. Can we fix this to say what isn't happening corrrectly?